include/onnxruntime/core/session/onnxruntime_c_api.h

```diff
@@ -438,12 +439,19 @@ typedef struct OrtCUDAProviderOptions {
    */
   OrtArenaCfg* default_memory_arena_cfg;
 
-  /** \brief Enable TunableOp.
-   *   Set it to 1 to enable TunableOp. Otherwise, it is disabled by default.
+  /** \brief Enable TunableOp for using.
+   *   Set it to 1/0 to enable/disable TunableOp. Otherwise, it is disabled by default.
    *   This option can be superseded by environment variable ORT_CUDA_TUNABLE_OP_ENABLED.
    */
   int tunable_op_enabled;
 
+  /** \brief Enable TunableOp for tuning.
+   *   Set it to 1/0 to enable/disable TunableOp tuning. Otherwise, it is disabled by default.
+   *   This option can be superseded by environment variable ORT_CUDA_TUNABLE_OP_TUNING.
```

Comment:
> superseded

Overridden might be a better choice of word here.

Reply:
Updated